### DISCUSSION TITLE
Set Bot Operators

### DISCUSSION THREAD

**Discussion by HeyMaxYT** [ScootKit/CustomDCBot#115] [Questions] [answered]
Hello my question is how can i set my self as Bot Operator i tried 
 "botOperators": [MyUserID],
when i restartet it said this:
![image](https://user-images.githubusercontent.com/86625067/225702640-77158905-3dba-43a9-82a4-f61b95ee4f7b.png)

**Comment by SCDerox** [comment]
This field only accepts strings, not int/bigint Data types (such as the one you entered).  If you don't want to worry about setup and configuration, we recommend using [SCNX](https://scnx.xyz/bot).

**Comment by HeyMaxYT** [comment, marked as answer]
when i understand it right it dosn´t work with this 725298084137140235 right ?

**Reply by DEVTomatoCake** [reply]
Use `"725298084137140235"` instead of `725298084137140235`.
